content/reference/cli/supported-commands.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cli Commands"
+---
+
+# Cli Commands
```

Comment:
Seems like a little blurb here about trying to be as similar to git as possible would be nice. Having a page that is just a list seems cold and unfreiendly